src/client.zig

```diff
@@ -70,12 +81,12 @@ pub const Client = struct {
             .request_timeout = .{
                 .name = "request_timeout",
                 .replica = std.math.maxInt(u16),
-                .after = 10,
+                .after = 1000,
             },
             .ping_timeout = .{
                 .name = "ping_timeout",
                 .replica = std.math.maxInt(u16),
-                .after = 10,
+                .after = 1000,
```

Comment:
`100` would be better here. This would be 1 second then if our tick frequency is every 10ms. Otherwise we might only retry after 10 seconds.